Tests/ViewInspectorTests/InspectorTests.swift

```diff
@@ -55,13 +55,19 @@ final class InspectorTests: XCTestCase {
         XCTAssertEqual(name2, "Struct1")
         let name3 = Inspector.typeName(value: Struct3<Int>(), generics: .remove)
         XCTAssertEqual(name3, "Struct3")
+        let name4 = Inspector.typeName(value: Struct3<(String) -> Void>(), generics: .remove)
+        XCTAssertEqual(name4, "Struct3")
     }
     
     func testTypeNameType() {
         let name1 = Inspector.typeName(type: Struct3<Int>.self)
         XCTAssertEqual(name1, "Struct3<Int>")
         let name2 = Inspector.typeName(type: Struct1.self)
         XCTAssertEqual(name2, "Struct1")
+        let name3 = Inspector.typeName(
+            type: ModifiedContent<EmptyView, _EnvironmentKeyWritingModifier<(String) -> Void>>.self,
```

Comment:
This is the same type that I saw in the original failure for the test that I added in EnvironmentModifiersTests.swift: https://github.com/nalexn/ViewInspector/pull/248/files#r1230245423